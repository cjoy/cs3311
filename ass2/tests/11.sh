# Enumerated program group
php members 115028
