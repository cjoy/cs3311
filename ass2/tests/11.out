5740
9200
9210
9230
